consensus-types/blocks/getters.go

```
@@ -80,6 +80,8 @@ func (b *SignedBeaconBlock) Copy() (interfaces.SignedBeaconBlock, error) {
 			return initBlindedSignedBlockFromProtoFulu(pb.(*eth.SignedBlindedBeaconBlockFulu).Copy())
 		}
 		return initSignedBlockFromProtoFulu(pb.(*eth.SignedBeaconBlockFulu).Copy())
+	case version.Gloas:
```

Not related to this line. But the getters file needs to be futher modified. Some getters do not make sense for Gloas and instead of returning nil and risking a panic they should return an error. For example the current path for `Execution()` will return `nil` instead of erroring out. The same for the `ExecutionRequests()`. We should add errors for all getters that would fail.